pyporscheconnectapi/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pyporscheconnectapi/cli.py,sha256=L9H0u3u5QXRzw3tGyq1PV3Zzws27eL5tyMD37GeJn78,6619
pyporscheconnectapi/client.py,sha256=-j3G9ONvXb5wBkw3bxNHWhnXbCApyWFfaEBAnDQM4dU,14430
pyporscheconnectapi/connection.py,sha256=udpCNr5aRIO-whBVOEm-7g5iWgGp86FvphhMbLI3szg,9879
pyporscheconnectapi/exceptions.py,sha256=Ez_D_TsafBd59LRIKRvGFv_6W9Ia425QCaClwXEke2w,1539
pyporscheconnectapi-0.0.14.dist-info/LICENSE,sha256=3wo4ylCH88NDNuSoNFaeaNCdzpQDloqsIUDQMg5FWec,1071
pyporscheconnectapi-0.0.14.dist-info/METADATA,sha256=ISE2nRecW_cJawnZE_X9jvhB8qvYmkPB64u_WhNLGQk,512
pyporscheconnectapi-0.0.14.dist-info/WHEEL,sha256=ewwEueio1C2XeHTvT17n8dZUJgOvyCWCt0WVNLClP9o,92
pyporscheconnectapi-0.0.14.dist-info/entry_points.txt,sha256=cq4djPuFhhopjtdIGqJ6w6vXOb6g65uLQSHnl7HAjfs,60
pyporscheconnectapi-0.0.14.dist-info/top_level.txt,sha256=YXSR3DO6I6QQ0lEQOvTts5BFIGsfOw_WtdFlD8BN4Yw,20
pyporscheconnectapi-0.0.14.dist-info/RECORD,,
